sktwocnt/lib/twocnt.f90

```diff
@@ -253,7 +253,7 @@ subroutine get_twocenter_integrals(inp, imap, skham, skover)
       call xc_f03_func_init(xcfunc_c, XC_GGA_C_PBE, XC_UNPOLARIZED)
     elseif (inp%iXC == 7) then
       call xc_f03_func_init(xcfunc_xc, XC_HYB_GGA_XC_B3LYP, XC_UNPOLARIZED)
-      call xc_f03_func_set_ext_params(xcfunc_xc, [0.20_dp, 0.72_dp, 0.81_dp])
+      call xc_f03_func_set_ext_params(xcfunc_xc, [inp%camAlpha, 0.72_dp, 0.81_dp])
```

Comment:
Probably needs a comment, as there are still hard coded values on the line.

Reply:
How about something like: Alpha is adjustable for B3LYP, but X and Y are from ...

Reply:
It is not yet adjustable from the user's perspective of course, but yes for this routine it effectively is. The other values are just the standard parametrization of B3LYP, I can add a reference to the paper.